### PR TITLE
v1

### PR DESCRIPTION
**Pull request by NistorCristian**
I've created 2 landing pages for Status v1 launch. Normal + korean.
I've replaced some images and changed some texts based on @j-zerah request.
Replaced download buttons with App Store, Google Play and Android.

This shoudn't be merged into master until @j-zerah approves.

### CODE REVIEW

File: themes/navy/languages/en.yml [hunk 1]
```diff
@@ -552,6 +557,24 @@ site:
...
+        title: "Truly Private Messaging"
+        description: "Send and receive fully encrypted messages without any central servers."
+      secure-wallet:
+        title: "Secure Crypto Wallet"
+        description: "Safely store your ETH, SNT, and other crypto-assets."
+      web3-browser:
+        title: "Decentralized Web3 Browser"
+        description: "Discover the ecosystem of decentralized apps, all together in one place."
+    korean:
+      title: "Join us in the public channel <a href='https://get.status.im/chat/public/korean' target='_blank'>#korean</a> and learn how you can win some SNT"
```
Comment (jakubgs) [inline themes/navy/languages/en.yml]: The channel is actually `#status-korean`.

File: source/v1-korean/index.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Private, Secure Communication
+description: Status brings the power of Ethereum into your pocket by combining a messenger, crypto-wallet, and Web3 browser.
+layout: v1-korean
```
Comment (jakubgs) [inline source/v1-korean/index.md]: Why make a separate page with a separate layout? I get that `v1-korean` includes the `div.korean-channel` with the message about the channel, but why not just generalize that for all languages we have?

You can put the links to the channels in `source/_data/channels.yml`, and load them into `div.channel-invite` based on language selected. For general english we can just use the `#status` channel, and for languages we can do `#status-${language}`.

File: themes/navy/languages/en.yml (hunk 1 above)
Comment (NistorCristian) [reply themes/navy/languages/en.yml]: I removed `package-lock.json`. The footer is ok. That is how the designer made it.

File: source/v1-korean/index.md (hunk 2 above)
Comment (NistorCristian) [reply source/v1-korean/index.md]: I've created `channels.yml` that does exactly what you said. I'm not sure if those are the right channels.

File: themes/navy/layout/partial/footer.ejs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<footer>
+<footer class="<% if (page.layout == 'v1' || page.layout == 'v1-korean') { %>grey<% } %>">
```
Comment (jakubgs) [inline themes/navy/layout/partial/footer.ejs]: I think this is obsolete now.

File: source/v1-korean/index.md (hunk 2 above)
Comment (jakubgs) [inline source/v1-korean/index.md]: This file is obsolete now.

File: themes/navy/languages/en.yml [hunk 4]
```diff
@@ -552,6 +557,24 @@ site:
       link-2: "Linux App"
       link-3: "Windows App"
       announcement: "This is alpha software and may contain vulnerabilities. <br>Please expect changes."
+  lp:
```
Comment (jakubgs) [inline themes/navy/languages/en.yml]: What does `lp` stand for? Not very descriptive.

File: source/v1-korean/index.md (hunk 2 above)
Comment (jakubgs) [reply source/v1-korean/index.md]: Nice.

File: themes/navy/languages/en.yml (hunk 1 above)
Comment (jakubgs) [reply themes/navy/languages/en.yml]: Thx.

**Comment by jakubgs** [comment]
Looks good: https://dev.status.im/v1/
Same for other languages like: https://dev.status.im/ko/v1/

I assume we don't want this pushed to Prod just yet. We still need translations and updated to a bunch of links, so I'm leaving this in `develop` for now.

**Comment by NistorCristian** [comment]
Yes, this shoudn't be live until @j-zerah gives his ok. He will provide the links when the app is approved on AppStore and Play Store.

**Pull request merged**